src/components/organisms/Badges/BadgeImage.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback, useRef } from "react";
+
+const defaultAvatar = "/icons/sparkle-nav-logo.png";
+
+export const BadgeImage: React.FC<{
+  image?: string;
+}> = ({ image }) => {
+  const imageRef = useRef<HTMLImageElement>(null);
+
+  const changeBadgeImage = useCallback(() => {
+    if (!imageRef.current) return;
+
+    imageRef.current.src = defaultAvatar;
+  }, []);
+
+  return (
+    <img
+      className="badge-list-item-image"
+      ref={imageRef}
+      src={image ?? defaultAvatar}
+      alt="Badge"
```

Comment:
@thecadams @0xdevalias This alt image is going to shown up if the image url fails to open. After that onError will be called and set the defaultAvatar. This will show the text "Badge" for a split second and that's why I removed the text and set it to empty string.

Reply:
Ha! That's a neat trick (this could potentially also be a solution/workaround for implementing Ed's suggestion of handling 'broken' images in chats)

Part of me wants to say that if we're going to set `alt` to an empty string, is there any point in us setting the attribute at all? Part of me wants to say that in the edgecase of a broken image showing up, does it even matter if we see the text for a split second? Alt text is often used for accessibility/screen reader/etc situations.. so I think I am leaning towards "if it shows up the name for a split second in the edge case where this occurs, that fine" to keep the accessibility/etc benefits.

Reply:
(so from my memory of the diff where I originally commented on this, I think keeping the venue title in this component is probably a good idea for that, so it can be "venueName badage")

Reply:
Yes, we can remove the alt image but then the eslint starts crying with the following message:
`img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.`

But this is a very very very edge case, where the venue is created but someone went in the firebase and deleted the image.

Reply:
We should leave it as it is.